Examples/RadarExample/app/build.gradle

```diff
@@ -15,5 +15,5 @@ android {
 dependencies {
   implementation 'com.android.support:appcompat-v7:27.1.1'
   implementation 'com.android.support:design:27.1.1'
-  implementation 'io.radar:sdk:2.0.0'
+  implementation 'io.radar:sdk:2.0.7+'
```

Comment:
@coolbrow `2.0.7+` or `2.0.+`?

Reply:
Both are valid, I just did this so we can guarantee a minimum of `2.0.7` if for some reason they have an earlier version cached or something.